src/scripts/index.js

```diff
@@ -1,10 +1,502 @@
+
+      newToDoIdDiv.appendChild(newToDoID); // add text node to child
+      newToDoNameDiv.appendChild(newToDoName);
+      newToDoDescDiv.appendChild(newToDoDesc);
+    }
+  } else {
+    console.log('No tickets to display');
+  }
+}
+
+displayColumns();
+displayTickets();
+
+
+const toDoColumn = document.querySelector('.toDo');
+
+if (toDoColumn !== null) {
+  toDoColumn.addEventListener('dblclick', function() {
+    console.log(toDoColumn);
+    console.log('apelez fct openmodal');
+    openModal();
+  });
+}
+
+
+// DRAGGABLE TODOS
+
+const draggableToDos = document.querySelectorAll('.newToDo');
+if (draggableToDos.length !== 0) {
+  for (const [i, draggableToDo] of draggableToDos.entries()) {
+    draggableToDo.addEventListener('dragstart', (e) => {
+      e.dataTransfer.setData('text/plain', draggableToDos[i].dataset.id);
+      // console.log(draggableToDos[i].dataset.id)
+    });
+  }
+
+
+  for (const dropZone of document.querySelectorAll('.content-column')) {
+    dropZone.addEventListener('dragover', (e) => {
+      e.preventDefault();
+      console.log('s-a tras elementul in noua zona');
+    });
+
+    dropZone.addEventListener('drop', (e) => {
+      console.log('elementul e lasat in noua zona');
+      e.preventDefault();
+      console.log(e);
+
+      const droppedElId = e.dataTransfer.getData('text/plain');
+      const droppedEl = document.querySelector(`[data-id="${droppedElId}"]`);
+
+      console.log(droppedEl);
+      console.log(droppedElId);
+      dropZone.appendChild(droppedEl);
+
+
+      const status =
+      droppedEl.parentNode.parentNode.firstElementChild.textContent;
+      updateTicketStatus(droppedElId, status);
+    });
+  }
+}
+
+
+/**
+   * Remember ticket column
+   *
+   * @param {number} droppedElId
+   * @param {string} status
+   */
+function updateTicketStatus(droppedElId, status) {
+  const allTickets = JSON.parse(localStorage.getItem('JiraTickets'));
+
+  for (const [index, ticket] of allTickets.entries()) {
+    if (ticket.id === `JIRA-${droppedElId}`) {
+      ticket.status = status;
+      allTickets.splice(index, 1);
+      allTickets.push(ticket);
+      break;
+    }
+  }
+
+  localStorage.setItem('JiraTickets', JSON.stringify(allTickets));
+}
+
+
+/**
+   * Add new column
+   *
+   */
+function addNewColumn() {
+  const titles = document.querySelectorAll('h4.title');
+  for (const [index, title] of titles.entries()) {
+    title.addEventListener('mouseenter', function(e) {
+      const addSpan = document.createElement('span');
+
+      console.log(title.textContent);
+
+      if (title.textContent !== 'To Do' && title.textContent !== 'Done') {
+        const addSpan2 = document.createElement('span');
+        addSpan2.classList.add('fas', 'fa-trash-alt');
+        title.appendChild(addSpan2);
+        const newCreatedSpanDelete =
+        document.querySelector('span.fas.fa-trash-alt');
+        newCreatedSpanDelete.addEventListener('click', function() {
```

Comment:
You can use "deleteColumn" directly as the second parameter, without invoking an anonymous function and call it inside that.